packages/slider/src/RangeSlider.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { space } from 'styled-system'
+import { deprecatedPropType } from 'pcln-design-system'
+
+import Range from 'rc-slider'
+import styleSlider from './styleSlider'
+
+const _RangeSlider = styleSlider(Range)
+
+const RangeSlider = (props) => {
+  return (
+    <_RangeSlider
```

Comment:
wow I didn't even know you could name a component like that and it still work in `<>`